lib/gds_api/publishing_api_v2.rb

```diff
@@ -92,10 +124,28 @@ def discard_draft(content_id, options = {})
     post_json!(discard_url(content_id), params)
   end
 
+  # FIXME: Add documentation
```

Comment:
These `FIXME`'s are for anyone to pick up, in exchange for internet points and lots of happy emoji.